src/js/api/core-shim.js

```diff
@@ -87,6 +88,7 @@ Object.assign(CoreShim.prototype, {
         const setupConfig = options;
         const configuration = Config(Object.assign({}, options), persisted);
         configuration.id = api.id;
+        configuration.language = getPlayerLanguage();
```

Comment:
Why set this here on the config? Does something else need to read this from anything other than the model? If not set this in the model, not the config.